modules/tag-input/test/spec.coffee

```diff
@@ -236,3 +236,23 @@ describe 'tag-input', ->
         mustMatchAutocomplete: false
       })
       expect(ti._.autocomplete.options.mustMatch).to.be.false
+
+    it 'should not add a tag when selecting the item via the autocomplete', ->
+      ti = new hx.TagInput(hx.detached('div').node(), {
+        autocompleteData: ['bob']
+      })
+      clock = sinon.useFakeTimers()
+      testHelpers.fakeNodeEvent(ti.input.node(), 'focus')({})
+      ti.input.value('b')
+      testHelpers.fakeNodeEvent(ti.input.node(), 'input')({})
+      clock.tick(dropdownAnimateDuration)
+
+      dropdown = ti._.autocomplete._.menu.dropdown._.dropdown
+      target = dropdown.select('.hx-menu-item').node()
+
+      testHelpers.fakeNodeEvent(ti.input.node(), 'blur')({})
+      testHelpers.fakeNodeEvent(dropdown.node(), 'click')({target: target})
+      ti.items().should.eql(['bob'])
```

Comment:
Tested this returns `['b','bob']` before the changes and realised that the dropdown event is fired in a weird way which made it awkward to test as you have to pass `target` through to the event. Might be useful if you need to test other menu-related things.